### PR TITLE
OpenID payload cache uses the wrong cache key

### LINKED ISSUE #1660: OpenID payload cache uses the wrong cache key
The `cache_key` becomes the same for every access token due to this bug: https://github.com/Kinto/kinto/blob/e1e0d6be0024418fd100210901f9d2ca06344fe1/kinto/plugins/openid/__init__.py#L51
No matter what the `hmac_tokens` variable is the `cache_key` always becomes `'openid:verify:%s'`.

### PR DESCRIPTION
**Pull request by peterbe**
Fixes #1660

- [ ] Add documentation.
- [x] Add tests.
- [x] Add a changelog entry.
- [x] Add your name in the contributors file.
- [ ] If you changed the HTTP API, update the [API_VERSION](https://github.com/Kinto/kinto/blob/master/kinto/__init__.py#L15) constant and add an API changelog entry [in the docs](https://github.com/Kinto/kinto/blob/master/docs/api/index.rst)
- [ ] If you added a new configuration setting, update the [`kinto.tpl`](https://github.com/Kinto/kinto/blob/master/kinto/config/kinto.tpl) file with it.

### CODE REVIEW

**Comment by peterbe** [comment]
I'm trying to figure out how to test this. Just trying to find my way around tox and venvs and stuff. Right now impatiently waiting for `make tests` to finish.

File: tests/plugins/test_openid.py [hunk 1]
```diff
@@ -111,13 +111,35 @@ def test_returns_none_if_no_cache_and_invalid_access_token(self):
     def test_payload_is_read_from_cache(self):
         self.request.headers['Authorization'] = 'Bearer xyz'
         self.request.registry.cache.get.return_value = {'sub': 'me'}
-        self.policy.unauthenticated_userid(self.request) == 'me'
+        assert self.policy.unauthenticated_userid(self.request) == 'me'
```
Comment (peterbe) [inline tests/plugins/test_openid.py]: This is unrelated but it stood out so I fixed it.

File: CONTRIBUTORS.rst [hunk 2]
```diff
@@ -57,6 +57,7 @@ Contributors
 * Oron Gola <oron.golar@gmail.com>
 * Palash Nigam <npalash25@gmail.com>
 * PeriGK <per.gkolias@gmail.com>
+* Peter Bengtsson <mail@peterbe.com>
```
Comment (leplatrem) [inline CONTRIBUTORS.rst]: :tada:

**Comment by peterbe** [comment]
What's the procedure for merging etc.?

**Comment by leplatrem** [comment]
> What's the procedure for merging etc.?

Once you have r+, and if you have permissions you can press the merge button :) We never took the time to setup bors or equivalent...

**Pull request merged**